CMakeLists.txt

```diff
@@ -37,7 +41,12 @@ macro(build_uncrustify)
   endif()
 
   include(ExternalProject)
-  find_package(Patch REQUIRED)
+
+  if(QNX)
+    find_host_package(Patch REQUIRED)
```

Comment:
I am surprised that this is necessary. In other cases where packages are cross compiled such a change doesn't seem to be the case. Why is the QNX case different from e.g. cross compiling for ARM?

Reply:
They should be both the same idea. Do you have a toolchain file for ARM you can share with me?
Here is the toolchain file we are using:
https://github.com/asobhy-qnx/ros2_root/blob/master/platform/qnx.nto.toolchain.cmake

Reply:
Please note the following: 
QNX_TARGET is where the target system libraries and binaries exist
QNX_STAGE is where the binary dependencies specific to ros2 exist
QNX_HOST is where QNX compiler and binutils exist

Reply:
Please see https://github.com/ros-tooling/cross_compile